generator/sbpg/targets/resources/sbp-cargo.toml

```diff
@@ -33,6 +33,11 @@ log = "0.4"
 nom = "6.0"
 thiserror = "1.0"
 
+[dependencies.swiftnav-rs]
+git = "https://github.com/swift-nav/swiftnav-rs"
+branch = "steve/sbp"
```

Comment:
are we waiting to get https://github.com/swift-nav/swiftnav-rs/pull/50 merged first?